src/templates.rs

```diff
@@ -9,19 +9,18 @@ use axum::response::{Html, IntoResponse, Response};
 
 #[derive(Template)]
 #[template(path = "idx.html")]
-pub struct IdxTemplate<'a> {
-    pub toml: &'static str,
-    pub examples: &'a [&'static str],
-    pub user: bool,
-    pub star: bool,
-    pub size: usize,
```

Comment:
we may want to keep this to validate the size of the upload on the client

Reply:
let's add it when we need it